scripts/opensearch-onetime-setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+# This script performs one-time setup for the OpenSearch tarball distribution.
+# It installs a demo security config and sets up the performance analyzer
+
+OPENSEARCH_HOME=`dirname $(realpath $0)`; cd $OPENSEARCH_HOME
```

Comment:
I think it would be better to ask the caller to specify OPENSEARCH_HOME, otherwise this script is too tightly coupled with the `deploy_cluster_tar_single.sh` script, and the user *must* place this script in the root of their installation. I'm not sure we want that.

Specify OPENSEARCH_HOME in the `deploy_cluster_tar_single.sh` script and export it to this one, and make this script fail with an error if OPENSEARCH_HOME is unset.

Reply:
Need more debug and possible changes on docker then.